test/test_storage_summary.py

```diff
@@ -0,0 +1,80 @@
+import unittest
+from apel.db.records.record import InvalidRecordException
+from apel.db.records.storage_summary import StorageSummaryRecord
+
+
+class StorageSummaryRecordTest(unittest.TestCase):
+    """
+    Test case for StorageSummaryRecord
+    """
+
+    def test_load_from_tuple(self):
+        # full example
+        data = ('MySite', 'host.example.org', 'pool-003', 'disk',
+                '/home/projectA', 'binarydataproject.example.org',
+                '2023-10-11T09:41:40Z', 10, 2023, 42, 14728, 13617, 0)
+
+        record = StorageSummaryRecord()
+        record.load_from_tuple(data)
+
+        self.assertEquals(record.get_field('Site'), 'MySite')
```

Comment:
As of Python 2.7, assertEquals() is being deprecated in favour of assertEqual() - https://docs.python.org/2/library/unittest.html#deprecated-aliases

Reply:
Cool, Updated the code. I think it would be good chance for us to have a ticket for this please ? what's say ?

Reply:
Could do. Wait for Adrian/Greg to confirm they would want it though